File: II - Linguaggi di programmazione e laboratorio/Esame/Python/03_compito_java_20150216/Cd.py
from Prodotto import Prodotto


class Cd(Prodotto):
    def __init__(self, tipo, codice, titolo, autore, prezzo, durata):
        super().__init__(tipo, codice, titolo, autore, prezzo)
        self._durata = durata

    def __str__(self):
        return super().__str__() + "\t-\t" + str(self._durata)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/03_compito_java_2015_02_16/Cd.java
import java.util.*;
import java.io.*;

public class Cd extends Prodotto {
    private int durata;

    public Cd(int codice, String titolo, String autori, int prezzo, int durata){
        super(codice, titolo, autori, prezzo);
        this.durata = durata;
    }
    public String toString() {
        return "cd\t" + codice + "\t" + titolo + "\t" + autori + "\t" + prezzo + "\t" +  "\t-\t" + durata;
    }
}